package.json

```diff
@@ -6,7 +6,9 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "bootstrap": "^4.5.3",
     "react": "^17.0.1",
+    "react-bootstrap": "^1.4.3",
```

Comment:
is there a difference between `bootstrap` and `react-bootstrap`? Are both needed to bring in the bootstrap components?

Reply:
the bootstrap is not needed. I have deleted it.